bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,81 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed, HTTPException
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+API_ROOT = "https://realpython.com/search/api/v1/?"
```

Comment:
Wait is the `?` necessary here?

Reply:
I'll need to double-check that on my test server before removing, but if it is not necessary I'll take it out.